src/test/scala/cromwell/util/SampleWdl.scala

```diff
@@ -12,6 +14,19 @@ trait SampleWdl {
   def wdlJson: WdlJson = {
     "{" + rawInputs.collect { case (k, v) => s""" "$k": "$v"""" }.mkString(",\n") + "}"
   }
+
+  def createCannedFile(prefix: String, contents: String, dir: Option[Path] = None): File = {
+    val suffix = ".out"
+    val file = dir match {
+      case Some(path) => Files.createTempFile(path, prefix, suffix)
```

Comment:
Not a critique of your code as I don't think it is reasonably resolvable in scala (but I wish it was!) but this is 2x in the same PR with the pattern of a some/none calling the same function with a different arity. It'd be nice it there was some way to smooth that out. W/ a pure scala function you could do that via currying but alas. Maybe hussein is right and scala is a pile of poop :)